### ISSUE TITLE
Manually overriding the buffer dependency to a later version?

### ISSUE THREAD

**Issue by rokkamokka** [jspm/nodelibs-buffer#5]
I apologize in advance because I realize this is probably not the proper place to ask this question.

I'm using JSPM 0.16 for a project, which installs jspm/nodelibs-buffer@0.1.0, which in turn installs npm:buffer@3.6.0.

However, npm:buffer@3.6.0 causes an issue with github:crypto-browserify/createHmac that causes it to generate improper hmacs.

I have confirmed that this issue disappears if I manually change the nodelibs-buffer npm:buffer@3.6.0 dependency to npm:buffer@5.0.6 in my generated config.js

However, I seem unable to perform this override in my package.json. I have tried adding both
```
    "overrides": {
      "github:jspm/nodelibs-buffer@0.1.0": {
        "dependencies": {
          "buffer": "^5.0.6"
        }
      }
    }
```
and
```
    "overrides": {
      "github:jspm/nodelibs-buffer@0.1.0": {
        "jspmNodeConversion": false,
        "dependencies": {
          "buffer": "npm:buffer@5.0.6"
        }
      }
    }
```

but neither seems to produce a change in my generated config.js. Am I going about this wrong? Is it even possible to override a dependency this way?

**Comment by guybedford** [comment]
Overrides require a `jspm install` to fully apply. I've published a 0.1.1 of this project with the update, let me know if that helps.

**Comment by rokkamokka** [comment]
I did try to `jspm install` between package.json changes, but nothing seemed to change in config.js (although the console output did say `warn Using local override for github:jspm/nodelibs-buffer@0.1.0`).

Thanks a bunch for the release anyway, it seems to have solved my problem!
